toolchain/sem_ir/import_ir.h

```diff
@@ -14,18 +14,20 @@ namespace Carbon::SemIR {
 // A reference to an imported IR.
 struct ImportIR : public Printable<ImportIR> {
   auto Print(llvm::raw_ostream& out) const -> void {
-    out << "{node_id: " << node_id
+    out << "{decl_id: " << decl_id
         << ", is_export: " << (is_export ? "true" : "false") << "}";
   }
 
-  // The node ID for the import.
-  Parse::ImportDeclId node_id;
-  // The imported IR.
-  const File* sem_ir;
+  // The `import` declaration.
+  InstId decl_id;
```

Comment:
Maybe `import_decl_id`?

Reply:
The equivalents I would draw would be `decl_id` on `AssociatedEntity`, or on non-instruction types, `Function` and `Generic`.